src/mpi/romio/adio/ad_quobytefs/Makefile.mk

```diff
@@ -0,0 +1,27 @@
+## -*- Mode: Makefile; -*-
+## vim: set ft=automake :
```

Comment:
Please remove these modelines.  We recently went through all files and removed them.